### PR TITLE
New CLI test for building compose with SELinux in enforcing mode

### PR DESCRIPTION
**Pull request by jikortus**
New test to check that composer can work with SELinux in enforcing mode on the host system.

### CODE REVIEW

**Comment by atodorov** [comment]
Note: for rhel7-extras we need #525 to be merged

**Comment by jikortus** [comment]
OK, so I just used a check for selinux mode instead of a separate test to save time and resources.

I wonder if there could be any downsides with putting a selinux mode check in the test execution loop in `tests/test_cli.sh`? At least it wouldn't be necessary to make sure the check is present in every (new) test created, so this might be a more consistent approach.

File: tests/cli/test_compose_ext4-filesystem.sh [hunk 1]
```diff
@@ -15,6 +15,10 @@ CLI="./src/bin/composer-cli"
 
 
 rlJournalStart
+    rlPhaseStart "SELinux operates in enforcing mode"
+        rlAssertEquals "$(getenforce)" "Enforcing"
```
Comment (atodorov) [inline tests/cli/test_compose_ext4-filesystem.sh]: `rlAssertEquals "SELinux operates in enforcing mode" "$(getenforce)" "Enforcing"`, see:
https://github.com/beakerlib/beakerlib/wiki/man#rlassertequals

and you can place the assert right after "compose start" opening clause. No need for separate phase only for that.

**Comment by atodorov** [comment]
> I wonder if there could be any downsides with putting a selinux mode check in the test execution loop in tests/test_cli.sh? At least it wouldn't be necessary to make sure the check is present in every (new) test created, so this might be a more consistent approach.

That would be the easiest thing to do but it skrews up the nice results formatting that we get from beakerlib. IMO having easy to understand failure messages will be more important in the future b/c we won't have the time luxury to figure out what went wrong when it does.

File: tests/cli/test_compose_ext4-filesystem.sh (hunk 1 above)
Comment (jikortus) [reply tests/cli/test_compose_ext4-filesystem.sh]: Thank you for pointing this out. Fixed.

**Comment by atodorov** [comment]
Jenkins failures reported in https://bugzilla.redhat.com/show_bug.cgi?id=1672971, not related to current changes.

**Pull request merged**